README.md

```
@@ -1,5 +1,82 @@
+  - [1. Nuclei CLI](#1-nuclei-cli)
+  - [2. Pro and Enterprise Editions](#2-pro-and-enterprise-editions)
+- [Documentation](#documentation)
+  - [Command Line Flags](#command-line-flags)
+  - [Single target scan](#single-target-scan)
+  - [Scanning multiple targets](#scanning-multiple-targets)
+  - [Network scan](#network-scan)
+  - [Scanning with your custom template](#scanning-with-your-custom-template)
+  - [Connect Nuclei to ProjectDiscovery](#connect-nuclei-to-projectdiscovery)
+- [Nuclei Templates, Community and Rewards 💎](#nuclei-templates-community-and-rewards-)
+- [Our Mission](#our-mission)
+- [Contributors ❤️](#contributors-️)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)